website/static/js/contribAdder.js

```diff
@@ -123,13 +133,19 @@ var AddContributorViewModel = oop.extend(Paginator, {
             return $.getJSON(
                 '/api/v1/user/search/', {
                     query: self.query(),
-                    excludeNode: nodeId,
                     page: self.pageToGet
                 },
                 function(result) {
                     var contributors = result.users.map(function(userData) {
                         return new Contributor(userData);
                     });
+                    for(var i = 0; i < contributors.length; i++) {
+                        if (self.contributors().indexOf(contributors[i].id) !== -1) {
+                            contributors[i].added = true;
```

Comment:
Rather than looping over contributors again, can you do this in the `map` call above?